src/screens/Dashboard/Footer/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebook, faLinkedin, faInstagram, faTwitter, faYoutube
+} from "@fortawesome/free-brands-svg-icons";
+import './style.scss';
+
+export default () => (
+  <section id="footer">
```

Comment:
Here we could use `<footer> ` instead of section for sematic HTML

Reply:
updated the tag